.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+    "extends": "cesium/browser"
```

Comment:
For Cesium, instead of using the npm published version of the tool, perhaps we should use a local file reference here, since it's in the same repository and would be easier to update that way.